templates/apiary-bucket-policy.json

```
@@ -83,7 +83,10 @@
                 "AWS": "*"
             },
             "Action": "s3:*",
-            "Resource": "arn:aws:s3:::${bucket_name}",
+            "Resource": [
+                "arn:aws:s3:::${bucket_name}",
+                "arn:aws:s3:::${bucket_name}/*",
```

Is IAM ok with comma in the last item in a list?  I can never remember.
```suggestion
                "arn:aws:s3:::${bucket_name}/*"
```